packages/buttons/stories/7-SplitButton.stories.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { action } from '@storybook/addon-actions';
+import { Story, Meta } from '@storybook/react';
+import { SplitButton, Button, ChevronButton } from '@zendeskgarden/react-buttons';
+import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Buttons/SplitButton',
+  component: SplitButton
+} as Meta;
+
+const StyledRow = styled(Row)`
+  padding: ${props => props.theme.space.xxl} 0;
+`;
+
+export const Default: Story = () => {
+  const [isOpen, setIsOpen] = useState<boolean | undefined>(false);
+
+  return (
+    <Grid>
+      <StyledRow>
+        <Col textAlign="center">
+          <SplitButton>
```

Comment:
Please provide the controls seen under the "Group" section of https://zendeskgarden.github.io/react-components/buttons/#basic.